packages/webcomponents/src/components/payout-details/payout-details-core.tsx

```diff
@@ -7,10 +7,12 @@ import StyledHost from '../../utils/styled-host/styled-host';
 
 @Component({
   tag: 'payout-details-core',
+  styleUrls: ['../../styles/icons.css'],
```

Comment:
@jakemer should I move this file to the root.css file?

Reply:
I don't think it'll work to move these styles to root.css because it needs a way (such as a css part) to make it through to the shadow DOM. I am wondering, though, if it would make sense to have an actual SVG file with the dimensions set on it and import it directly into the component rather than using css/background-image? Or is there a bootstrap native way to do icons?